packages/theme/pages/Checkout/Customer.vue

```diff
@@ -128,15 +131,20 @@ export default {
     });
 
     const handleFormSubmit = async () => {
-      await $vendure.api.setCustomerForOrder(form.value);
+      const response = await $vendure.api.setCustomerForOrder(form.value);
```

Comment:
Maybe a better option for sending a form is to make a composable, instead of directly calling api-client function

Reply:
I will create an issue for that. This would require a new composable and factory and I will develop it for stable 1.0.0 version. For now we would release this MVP as 1.0.0-rc.1